### PR TITLE
Add missing translations

### PR DESCRIPTION
**Pull request by VincentLanglet**
## Subject

I am targeting this branch, because BC.

## Changelog

```markdown
### Added
- Added `ar` translations for btn_advanced_filters, link_edit, title_show, flash_lock_error, flash_batch_no_elements_processed, form_not_available, td_compare, td_role, label_compare_revision, list_results_count_prefix, noscript_warning, message_form_group_empty, link_filters, stats_view_more, title_select_subclass, no_subclass_available, label_unknown_user, read_more, read_less, toggle_navigation.
- Added `bg` translations for link_edit, title_show, flash_batch_no_elements_processed, read_more, read_less, toggle_navigation.
- Added `ca` translations for link_edit, flash_batch_no_elements_processed, toggle_navigation.
- Added `cs` translations for sonata_administration, link_edit, title_show, admin, flash_batch_no_elements_processed, td_role, short_object_description_placeholder, title_select_subclass, no_subclass_available, label_unknown_user, read_more, read_less, toggle_navigation.
- Added `de` translations for sonata_administration, admin, flash_batch_no_elements_processed.
- Added `es` translations for flash_batch_no_elements_processed.
- Added `eu` translations for sonata_administration, btn_advanced_filters, link_edit, title_show, flash_lock_error, flash_batch_no_elements_processed, form_not_available, td_compare, td_role, label_compare_revision, list_results_count_prefix, short_object_description_placeholder, title_search_results, search_placeholder, no_results_found, add_new_entry, link_actions, noscript_warning, message_form_group_empty, link_filters, stats_view_more, title_select_subclass, no_subclass_available, label_unknown_user, read_more, read_less, toggle_navigation.
- Added `fa` translations for link_edit, title_show, flash_batch_no_elements_processed, label_unknown_user, read_more, read_less, toggle_navigation.
- Added `hr` translations for btn_advanced_filters, link_edit, title_show, Admin, flash_lock_error, flash_batch_no_elements_processed, form_not_available, td_role, list_results_count_prefix, link_filters, stats_view_more, title_select_subclass, no_subclass_available, label_unknown_user, read_more, read_less, toggle_navigation.
- Added `hu` translations for link_edit, flash_batch_no_elements_processed, label_unknown_user, read_more, read_less, toggle_navigation.
- Added `it` translations for read_more, read_less, toggle_navigation.
- Added `ja` translations for btn_advanced_filters, link_edit, title_show, flash_lock_error, flash_batch_no_elements_processed, form_not_available, td_compare, td_role, label_compare_revision, list_results_count_prefix, toggle_navigation.
- Added `lb` translations for sonata_administration, td_compare.
- Added `lt` translations for btn_advanced_filters, link_edit, title_show, flash_lock_error, flash_batch_no_elements_processed, form_not_available, td_compare, td_role, label_compare_revision, list_results_count_prefix, noscript_warning, message_form_group_empty, link_filters, stats_view_more, title_select_subclass, no_subclass_available, label_unknown_user, read_more, read_less, toggle_navigation.
- Added `lv` translations for link_edit, flash_batch_no_elements_processed, toggle_navigation.
- Added `no` translations for link_edit, title_show, flash_lock_error, flash_batch_no_elements_processed, form_not_available, stats_view_more, label_unknown_user, read_more, read_less, toggle_navigation.
- Added `pl` translations for flash_lock_error, flash_batch_no_elements_processed, td_compare, td_role, label_compare_revision, toggle_navigation.
- Added `pt` translations for sonata_administration, btn_advanced_filters, link_edit, title_show, flash_lock_error, flash_batch_no_elements_processed, form_not_available, td_compare, td_role, label_compare_revision, list_results_count_prefix, label_export_download, confirm_exit, link_edit_acl, btn_update_acl, flash_acl_edit_success, link_action_acl, title_search_results, search_placeholder, no_results_found, add_new_entry, link_actions, noscript_warning, message_form_group_empty, message_form_group_empty, stats_view_more, title_select_subclass, no_subclass_available, label_unknown_user, read_more, read_less, toggle_navigation.
- Added `pt_BR` translations for link_edit flash_batch_no_elements_processed, label_export_download, read_more, read_less, toggle_navigation.
- Added `ro` translations for sonata_administration, link_edit, title_show, flash_lock_error,  flash_batch_no_elements_processed, form_not_available, td_compare, td_role, label_compare_revision, list_results_count_prefix, stats_view_more, title_select_subclass, no_subclass_available, label_unknown_user, read_more, read_less, toggle_navigation.
- Added `ru` translations for sonata_administration.
- Added `sk` translations for sonata_administration, link_edit, title_show, flash_batch_no_elements_processed, td_role, short_object_description_placeholder, title_select_subclass, no_subclass_available, label_unknown_user, read_more, read_less, toggle_navigation.
- Added `sl` translations for link_edit, title_show, flash_batch_no_elements_processed, read_more, read_less, toggle_navigation.
- Added almost all `sv_SE` translations.
- Added `tr` translations for link_edit, flash_batch_no_elements_processed, toggle_navigation.
- Added `uk` translations for sonata_administration, link_edit, flash_batch_no_elements_processed, read_more, read_less, toggle_navigation.
- Added `zn_CH` translations for link_edit, title_show, flash_lock_error, flash_batch_no_elements_processed, form_not_available, td_compare, td_role, label_compare_revision, list_results_count_prefix, stats_view_more, title_select_subclass, no_subclass_available, label_unknown_user, read_more, read_less, toggle_navigation.
```

### CODE REVIEW

File: src/Resources/translations/SonataAdminBundle.es.xliff [hunk 1]
```diff
@@ -178,6 +178,10 @@
                 <source>flash_batch_delete_success</source>
                 <target>Los elementos seleccionados han sido eliminados con éxito.</target>
             </trans-unit>
+            <trans-unit id="flash_batch_no_elements_processed">
+                <source>flash_batch_no_elements_processed</source>
+                <target>No hay artículos procesados.</target>
```
Comment (jordisala1991) [inline src/Resources/translations/SonataAdminBundle.es.xliff:183]: No se han procesado elementos.

Comment (VincentLanglet) [reply src/Resources/translations/SonataAdminBundle.es.xliff]: Thanks @jordisala1991, done.

File: src/Resources/translations/SonataAdminBundle.pl.xliff [hunk 2]
```diff
@@ -324,7 +328,7 @@
             </trans-unit>
             <trans-unit id="td_compare">
                 <source>td_compare</source>
-                <target>td_compare</target>
+                <target>Porównać</target>
```
Comment (wbloszyk) [inline src/Resources/translations/SonataAdminBundle.pl.xliff:331]:
```suggestion
                <target>Porównaj</target>
```
It something like 'You be'

Comment (VincentLanglet) [reply src/Resources/translations/SonataAdminBundle.pl.xliff]: Thanks, it's fixed !

**Comment by core23** [comment]
I appreciate your work, but where did you find the translations for all this different languages? Can you confirm, that they are all correct in the used context?

**Comment by VincentLanglet** [comment]
> I appreciate your work, but where did you find the translations for all this different languages? Can you confirm, that they are all correct in the used context?

Can you confirm that the existing translations are correct in the used context ?

Currently missing translations are, either fallback to english, either fallback to dummy translations.
- For some of these translations, I used the existing ones. `Create %name%` is just `Edit %name%` where I replace `Edit` by the translation of `Create`, which was already in the translations.
- For others of these translations, I used a dictionary, a translator with approved translations, selected the word with was the most context-related, and try to reverse-translate the translations in order to check the means is the one wanted.

I can't confirm you that 100% of these translations are correct. I you find one which is not, I'll be happy to change it. But I can confirm that most of them are, and this is a far better situation than the current situation with the english fallback or the dummy translations, which are less understandable.

**Comment by core23** [comment]
> - Added missing translations.

Can you please list the translations in the Changelog

**Comment by VincentLanglet** [comment]
> > * Added missing translations.
> 
> Can you please list the translations in the Changelog

Done

**Comment by jordisala1991** [comment]
Thank you @VincentLanglet

**Comment by greg0ire** [comment]
Uuuuuh do we really need that level of detail in the changelog?

**Comment by core23** [comment]
> > > * Added missing translations.
> > 
> > 
> > Can you please list the translations in the Changelog
> 
> Done



> Uuuuuh do we really need that level of detail in the changelog?

IMHO a simple changelog would be enough:

```
- Update `cs` translation
...
```

**Comment by greg0ire** [comment]
Yeah plus the current changelog has a lot of unquoted underscored that have meaning in markdown.

**Comment by VincentLanglet** [comment]
> > * Added missing translations.
> 
> Can you please list the translations in the Changelog

Please be more clear next time if you want that I list the translation locale and not the translation

**Pull request merged**